Spotkanie3_Zadania/String/String/Program.cs

```diff
@@ -0,0 +1,267 @@
+                Console.WriteLine("\nFirst select the operation you want to perform on the string:");
+                SelectOptions();
+                EndProgramme();
+            }
+        }
+
+        static void SelectOptions()
+        {
+            bool rightNumber = true;
+            
+            Console.WriteLine("1: For print your word.");
+            Console.WriteLine("2: Inverted word.");
+            Console.WriteLine("3: For concatenate string using 'String.Join'.");
+            Console.WriteLine("4: If you want to enlarge letters in your word.");
+            Console.WriteLine("5: If you want to change the letters in your word to small ones.");
+            Console.WriteLine("6: If you want the number of letters in your word.");
+            Console.WriteLine("7: If you want to write each letter separately.");
+            Console.WriteLine("8: For print the same sentance in diffrent way.");
+            Console.WriteLine("9: If you want to compare words.");
+            Console.WriteLine("10: For print your word.");
+            Console.Write("\nYou choose: ");
+            
+            while(true)
+            {
+                switch (EnterNumber())
+                {
+                    case 1:
+                        Console.WriteLine($"Your word is: {TakeaWord()}");
+                        break;
+                    case 2:
+                        Console.WriteLine(ReverseTheWord(TakeaWord()));
+                        break;
+                    case 3:
+                        Console.WriteLine(ConcatenateWords(ChangeWordToArray(TakeaWord())));
+                        break;
+                    case 4:
+                        Console.WriteLine(EnlargeLetters(TakeaWord()));
+                        break;
+                    case 5:
+                        Console.WriteLine(LettersSmaller(TakeaWord()));
+                        break;
+                    case 6:
+                        Console.WriteLine(CountNumberOfLettersInGivenWord(TakeaWord()));
+                        break;
+                    case 7:
+                        SpellWord(TakeaWord());
+                        break;
+                    case 8:
+                        ConcatenateString(TakeaWord());
+                        break;
+                    case 9:
+                        CompareWord(TakeaWord(), TakeaWord());
+                        break;
+                    case 10:
+                        ReturnsSubstringOfString(TakeaWord());
+                        break;
+                    default:
+                        Console.Write("The selected number is not from 1 to 10. Please try again: ");
+                        rightNumber = false;
+                        break;
+                }
+                if (rightNumber) break;
+                else rightNumber = true;
+                
+            }
+            
+        }
+
+        static string TakeaWord()
```

Comment:
```suggestion
        static string GetWord()
```

`TakeWord` tez mogloby byc, ale Get jest taki bardziej naturalny w programowaniu.
Uzywanie przedimkow czyli `a, an, the` w nazwach jest raczej zbędne, chyba, że faktycznie bardzo chcemy być spójni, np w nazwą elementu z aplikacji.
Raczej bym unikał dodawania przedimkow do nazw.